<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game</string>
  </property>
  <widget class="QPushButton" name="But1">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>420</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft PhagsPa</family>
     <pointsize>20</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
    /* 分层背景：图片在下，渐变在上 */
    background-image: url(:/res/ground.jpg),
                     qradialgradient(spread:repeat,
                                    cx:0.5, cy:0.5, radius:0.077,
                                    fx:0.5, fy:0.5,
                                    stop:0 rgba(0, 169, 255, 58),
                                    stop:0.5 rgba(0, 0, 0, 58),
                                    stop:1 rgba(0, 169, 255, 58));
    
    /* 背景混合模式 */
    background-blend-mode: overlay;
    
    /* 备用背景色 */
    background-color: rgb(85, 170, 127);
    
    /* 字体设置 */
    font: 20pt &quot;Microsoft PhagsPa&quot;;
}</string>
   </property>
   <property name="text">
    <string>返回上一页</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>0</y>
     <width>261</width>
     <height>231</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#5500ff;&quot;&gt;按&lt;/span&gt;&lt;span style=&quot; font-size:14pt; color:#ffff00;&quot;&gt;WASD键&lt;/span&gt;&lt;span style=&quot; font-size:14pt; color:#5500ff;&quot;&gt;来控制人物&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#5500ff;&quot;&gt;的移动&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#5500ff;&quot;&gt;其中&lt;/span&gt;&lt;span style=&quot; font-size:14pt; color:#ffff00;&quot;&gt;星星&lt;/span&gt;&lt;span style=&quot; font-size:14pt; color:#5500ff;&quot;&gt;为迷宫出口&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#5500ff;&quot;&gt;迷宫起点到终点可能&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#5500ff;&quot;&gt;没有通路，这个情况&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#5500ff;&quot;&gt;可以按下面的迷宫无解&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btu2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>340</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft PhagsPa</family>
     <pointsize>20</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
    /* 分层背景：图片在下，渐变在上 */
    background-image: url(:/res/ground.jpg),
                     qradialgradient(spread:repeat,
                                    cx:0.5, cy:0.5, radius:0.077,
                                    fx:0.5, fy:0.5,
                                    stop:0 rgba(0, 169, 255, 58),
                                    stop:0.5 rgba(0, 0, 0, 58),
                                    stop:1 rgba(0, 169, 255, 58));
    
    /* 背景混合模式 */
    background-blend-mode: overlay;
    
    /* 备用背景色 */
    background-color: rgb(85, 170, 127);
    
    /* 字体设置 */
    font: 20pt &quot;Microsoft PhagsPa&quot;;
}</string>
   </property>
   <property name="text">
    <string>自动寻路</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btu3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>260</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft PhagsPa</family>
     <pointsize>20</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
    /* 分层背景：图片在下，渐变在上 */
    background-image: url(:/res/ground.jpg),
                     qradialgradient(spread:repeat,
                                    cx:0.5, cy:0.5, radius:0.077,
                                    fx:0.5, fy:0.5,
                                    stop:0 rgba(0, 169, 255, 58),
                                    stop:0.5 rgba(0, 0, 0, 58),
                                    stop:1 rgba(0, 169, 255, 58));
    
    /* 背景混合模式 */
    background-blend-mode: overlay;
    
    /* 备用背景色 */
    background-color: rgb(85, 170, 127);
    
    /* 字体设置 */
    font: 20pt &quot;Microsoft PhagsPa&quot;;
}</string>
   </property>
   <property name="text">
    <string>迷宫无解</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
